mbed/mbed.py

```diff
@@ -2451,23 +2453,24 @@ def compile_(toolchain=None, target=None, profile=False, compile_library=False,
                   + args,
                   env=env)
 
+            if flash or sterm:
+                detected = program.detect_target()
+                try:
+                    from mbed_host_tests.host_tests_toolbox import flash_dev
+                except (IOError, ImportError, OSError):
+                    error("The '-f/--flash' option requires that the 'mbed-greentea' python module is installed.\nYou can install mbed-greentea by running 'pip install mbed-greentea'.", 1)
```

Comment:
This is not strictly true. The `-f/--flash` option requires `mbed-host-tests`.

Reply:
I'd rather encourage the user to install `mbed-greentea` which will pull in all the needed modules behind it. In this case, the product is `mbed-greentea`, not host-tests.

Reply:
Sure, that works. If `mbed-host-tests` is not something the user should ever install standalone, I'll merge it with greentea.

Reply:
In case this was not clean: this is resolved.